src/6model/reprs/P6bigint.c

```diff
@@ -92,10 +89,10 @@ static MVMint64 get_int(MVMThreadContext *tc, MVMSTable *st, MVMObject *root, vo
     if (MVM_BIGINT_IS_BIG(body)) {
         mp_int *i = body->u.bigint;
         if (MP_NEG == SIGN(i)) {
-            return -mp_get_int64(tc, i);
+            return -mp_get_int64(tc, i, 1);
```

Comment:
I think I have a slight preference for as is, but don't mind changing if there's consensus. But FWIW, my branch that attempts to figure out a solution to the more general uint problem ends up re-writing this so there's no `-` at all.